lib/solargraph/rbs_map/conversions.rb

```diff
@@ -229,14 +228,6 @@ def create_constant(name, tag, comments, type)
           closure: closure,
           comments: comments
         )
-        tag = if type == :class
-                "Class<#{tag}>"
-              elsif type == :module
-                "Module<#{tag}>"
-              else
-                Solargraph.logger.warn "Unrecognized constant type: #{type.inspect}"
-                "Class<#{tag}>"
-              end
```

Comment:
@apiology This tag adjustment appears to be redundant. The `generic_type` method already handles parameterized `Class` and `Module` types. I verified with a test in `rbs_map_spec.rb`, but figured I should bring it to your attention in case there's something I missed.

Reply:
Hmm. It looks like the use case I missed was Ruby 3.1 😞

Reply:
I think I got it. Aliases need the conversion to `Class<T>` or `Module<T>`, but constants do not.

Reply:
Ah, got it!